src/codeflare_sdk/templates/base-template.yaml

```diff
@@ -49,7 +49,7 @@ spec:
           # - kubernetes
         spec:
           # The version of Ray you are using. Make sure all Ray containers are running this version of Ray.
-          rayVersion: '2.7.0'
+          rayVersion: '2.20.0'
```

Comment:
Do we want to pin the Ray version in the container image tag as well?

Reply:
Actually my understanding is that Ray version field is rather pretty loose. It's only used for assembling the command for the Ray job submitter pod: https://github.com/ray-project/kuberay/blob/2c47c2d4d48be52bb6a2693d3aa850a763db5808/ray-operator/controllers/ray/common/job.go#L68.

Now on the point you're raising, I think it should be documented that the client and backend version must be compatible.